app/views/content/life-as-a-teacher/pay-and-benefits/_what-teachers-have-to-say.html.erb

```diff
@@ -8,22 +8,19 @@
             <ul>
               <%= render Content::InspirationCardComponent.new(heading_tag: "h3", card:
                 OpenStruct.new(
-                  title: "Teacher pay",
-                  description: "Find out the starting salary for a qualified teacher and how your pay could increase during your career.",
-                  path: "/life-as-a-teacher/pay-and-benefits/teacher-pay",
-                  image: "static/images/content/hero-images/teacher.jpg"
-                )) %>
-              <%= render Content::InspirationCardComponent.new(heading_tag: "h3", card:
-                OpenStruct.new(
-                  title: "Teacher pensions",
-                  description: "Learn more about the teachers' pension scheme which is one of the most generous in the country.",
-                  path: "/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme"
+                  title: "Ben's favourite things about teaching",
+                  description: "Maths and science teacher Ben shares his favourite things about teaching.",
+                  path: "/life-as-a-teacher/pay-and-benefits/bens-favourite-things-about-teaching",
+                  image: "static/images/content/case-studies/ben.jpg"
                 )) %>
+            </ul>
+            <ul>
               <%= render Content::InspirationCardComponent.new(heading_tag: "h3", card:
                 OpenStruct.new(
-                  title: "Teacher insights: moving up the career ladder",
-                  description: "Top tips from teacher training advisers on how you can progress a career in teaching.",
-                  path: "/life-as-a-teacher/pay-and-benefits/career-progression"
+                  title: "Abigail's favourite things about teaching",
```

Comment:
Are these two headings the same deliberately? `xxx's favourite things about teaching`

Not a problem if so - just wanted to check!

Reply:
Yes they are!